test/e2e/cluster/cfn-templates/setup-cfn.yaml

```diff
@@ -330,6 +330,69 @@ Resources:
                   - "|"
                   - !Ref PortWithDescription
 
+  HybridNodeDefaultSecurityGroupIngressFromProxyOnly:
+    Type: AWS::EC2::SecurityGroupIngress
+    Properties:
+      GroupId: !GetAtt HybridNodeVPC.DefaultSecurityGroup
+      IpProtocol: tcp
+      FromPort: 3128
+      ToPort: 3128
+      SourceSecurityGroupId: !Ref HybridNodeProxyOnlySecurityGroup
+
+  HybridNodeProxyOnlySecurityGroup:
+    Type: AWS::EC2::SecurityGroup
+    Properties:
+      GroupDescription: Security group for hybrid nodes that only allows traffic through the proxy
+      VpcId: !Ref HybridNodeVPC
+      SecurityGroupIngress:
+        - IpProtocol: -1
+          FromPort: -1
+          ToPort: -1
+          SourceSecurityGroupId: !GetAtt HybridNodeVPC.DefaultSecurityGroup
+      Tags:
+        - Key: Name
+          Value: !Sub ${ClusterName}-hybrid-node-proxy-sg
+        - Key: !Ref TestClusterTagKey
+          Value: !Ref ClusterName
+        - Key: !Ref CreationTimeTagKey
+          Value: !Ref CreationTime
+
+  HybridNodeProxyOnlySecurityGroupIngress10250:
+    Type: AWS::EC2::SecurityGroupIngress
+    Properties:
+      GroupId: !Ref HybridNodeProxyOnlySecurityGroup
+      IpProtocol: tcp
+      FromPort: 10250
+      ToPort: 10250
+      CidrIp: !Ref ClusterVPCCidr
+
+  HybridNodeProxyOnlySecurityGroupIngress9443:
+    Type: AWS::EC2::SecurityGroupIngress
+    Properties:
+      GroupId: !Ref HybridNodeProxyOnlySecurityGroup
+      IpProtocol: tcp
+      FromPort: 9443
+      ToPort: 9443
+      CidrIp: !Ref ClusterVPCCidr
+
+  HybridNodeProxyOnlySecurityGroupEgress:
+    Type: AWS::EC2::SecurityGroupEgress
+    Properties:
+      GroupId: !Ref HybridNodeProxyOnlySecurityGroup
+      IpProtocol: tcp
+      FromPort: 3128
+      ToPort: 3128
+      DestinationSecurityGroupId: !GetAtt HybridNodeVPC.DefaultSecurityGroup
+
+  HybridNodeProxyOnlySecurityGroupEgressToCluster:
+    Type: AWS::EC2::SecurityGroupEgress
```

Comment:
do we need to create a default egress block everything else? i thought the default is egress to everything